public/plugin.tsx

```diff
@@ -497,7 +504,8 @@ export class ObservabilityPlugin
       tableDetail: AssociatedObject,
       datasourceName: string,
       handleRefresh?: () => void,
-      dataSourceMDSId?: string
+      dataSourceMDSId?: string,
+      isS3ConnectionWithLakeFormation?: boolean
```

Comment:
Let's try to add datasource type here rather than a boolean to handle future coming datasources instaed of using specific bool: `isS3ConnectionWithLakeFormation?: boolean`

Reply:
Leaving it as is for now. Will revisit this in future as we try to align the UX for Flint and Security Lake